modules/dmrpp_module/data/get_dmrpp.in

```diff
@@ -64,31 +65,31 @@ BES.Data.RootDirectory=/dev/null
 BES.Catalog.catalog.TypeMatch=dmrpp:.*\.(dmrpp)$;
 BES.Catalog.catalog.TypeMatch+=h5:.*(\.bz2|\.gz|\.Z)?$;
 
-BES.Catalog.catalog.Include=;
-BES.Catalog.catalog.Exclude=^\..*;
+#jhrg BES.Catalog.catalog.Include=;
+#jhrg BES.Catalog.catalog.Exclude=^\..*;
```

Comment:
I think removing this is fraught. Did you test this with paths that include ".."? Like an end to end test with besstandalone. In general it is our policy to disallow ".." in any path.  Period.

Reply:
The rejection of .. in paths is in the C++ code. 

               if (dirEntry != "." && dirEntry != "..") {

BESFSDir.cc:120